RPIPs/RPIP-60.md

```diff
@@ -28,16 +28,18 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 
 ### Upgrade Delay
 - All protocol upgrades SHALL have an `upgrade_delay` delay between when they are passed and when they are executed
-- The `upgrade_delay` SHALL be a pDAO setting and its initial value SHALL be 3 weeks
+- The `upgrade_delay` SHALL be a pDAO setting and its initial value SHALL be 1 week
 
 
 ### Security Council Veto
 - The security council SHALL have the power to veto contract upgrades (which are proposed by the oDAO)
+  - `rocketDAOProtocolSettingsSecurity.upgradeveto.quorum` SHALL be a pDAO setting and its initial value SHALL be 33%
```

Comment:
I agree we need to set this somewhere. Should it be here, or should it defer to as yet not existing Security Council RPIP? I'm happy to just have a rationale item or something saying its here for now because the other thing doesn't exist, and instruct RPIP editors to move / remove it in the future if deemed appropriate?

Reply:
Feel free to disregard, more of a stylistic / structuring thing than anything.

Reply:
I think it should be here. I'm not against reorganizing at some point if helpful (the majority of the SC stuff is RPIP-33)